From: Armin Novak <armin.novak@thincast.com>
Date: Mon, 28 Mar 2022 11:52:52 +0200
Subject: Fixed windows build (_snpnrintf)

(cherry picked from commit 6cccc95691ae015e9f30121a3d70f9286724413f)
(cherry picked from commit c8dced590560fcf7dda974d4216c48197ad86de2)
---
 winpr/libwinpr/sspi/NTLM/ntlm.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/winpr/libwinpr/sspi/NTLM/ntlm.c b/winpr/libwinpr/sspi/NTLM/ntlm.c
index 98a130b..372e6fb 100644
--- a/winpr/libwinpr/sspi/NTLM/ntlm.c
+++ b/winpr/libwinpr/sspi/NTLM/ntlm.c
@@ -25,6 +25,7 @@
 #include <winpr/assert.h>
 #include <winpr/sspi.h>
 #include <winpr/print.h>
+#include <winpr/string.h>
 #include <winpr/tchar.h>
 #include <winpr/sysinfo.h>
 #include <winpr/registry.h>
@@ -1293,7 +1294,7 @@ char* ntlm_negotiate_flags_string(char* buffer, size_t size, UINT32 flags)
 	if (!buffer || (size == 0))
 		return buffer;
 
-	snprintf(buffer, size, "[0x%08" PRIx32 "] ", flags);
+	_snprintf(buffer, size, "[0x%08" PRIx32 "] ", flags);
 
 		for (x = 0; x < 31; x++)
 		{
